generators/customcontrol/index.js

```diff
@@ -0,0 +1,30 @@
+import chalk from "chalk"
+import Generator from "yeoman-generator"
+import prompts from "./prompts.js"
+import { lookForParentUI5ProjectAndPrompt } from "../helpers.js"
+
+export default class extends Generator {
+	static displayName = "Create a new xml view for an existing uimodule."
+
+	async prompting() {
+		await lookForParentUI5ProjectAndPrompt.call(this, prompts)
+	}
+
+	async writing() {
+		this.log(chalk.green(`✨ creating new custom control for ${this.options.config.uimoduleName}`))
+
+		const webappPath = `${this.options.config.uimoduleName}/webapp`
```

Comment:
just as a hint for this and all following path constructs: does `yo` take care about cross-platform paths? If so, ignore this. If not, you might have to refactor to `path.resolve(this.options.config.uimoduleName, "webapp")` for this and all similar path constructs

Reply:
Yes, not windows compatible atm, have to fix this.